settings/cronjob.ini.append.php

```diff
@@ -0,0 +1,12 @@
+<?php /*
+
+[CronjobSettings]
+ExtensionDirectories[]=ezplatformsearch
+
+[CronjobPart-frequent]
```

Comment:
Is it a good idea to add it to frequent?

I'd rather leave more control to users if this should appear in frequent part or not.